applications/simulate_prod.py

```diff
@@ -0,0 +1,346 @@
+    config.parser.add_argument(
+        "--production_config",
+        help=(
+            "Simulation configuration file "
+            "(contains the default setup which can be overwritten by the command line options)"
+        ),
+        type=str,
+        required=True,
+    )
+    config.parser.add_argument(
+        "--prod_tag",
+        help="The production tag (ID) to use (e.g., Prod5)",
+        type=str,
+        required=True,
+    )
+    config.parser.add_argument(
+        "--site",
+        help="CTAO site (e.g., Paranal or LaPalma, case insensitive)",
+        type=str.lower,
+        required=True,
+        choices=[
+            "paranal",
+            "lapalma",
+        ],
+    )
+    config.parser.add_argument(
+        "--primary",
+        help="Primary particle to simulate.",
+        type=str.lower,
+        required=True,
+        choices=[
+            "gamma",
+            "gamma_diffuse",
+            "electron",
+            "proton",
+            "muon",
+            "helium",
+            "nitrogen",
+            "silicon",
+            "iron",
+        ],
+    )
+    config.parser.add_argument(
+        "--from_direction",
+        help="Direction from which the primary reaches the atmosphere",
+        type=str.lower,
+        required=True,
+        choices=[
+            "north",
+            "south",
+            "east",
+            "west",
+        ],
+    )
+    config.parser.add_argument(
+        "--zenith_angle",
+        help="Zenith angle in degrees",
+        type=CommandLineParser.zenith_angle,
+        required=True,
+    )
+    config.parser.add_argument(
+        "--nshow",
+        help="Number of showers to simulate",
+        type=int,
+        required=False,
+    )
+    config.parser.add_argument(
+        "--start_run",
+        help=(
+            "Start run number such that the actual run number will be 'start_run' + 'run'. "
+            "This is useful in case a new transform is submitted for the same production. "
+            "It allows the transformation system to keep using sequential run numbers without "
+            "repetition."
+        ),
+        type=int,
+        required=True,
+    )
+    config.parser.add_argument(
+        "--run",
+        help="Run number (actual run number will be 'start_run' + 'run')",
+        type=int,
+        required=True,
+    )
+    config.parser.add_argument(
+        "--data_directory",
+        help="The directory where to save the corsika-data and simtel-data output directories.",
+        type=str.lower,
+        required=False,
+        default="./",
+    )
+    return config.initialize(db_config=True)
+
+
+def _proccess_simulation_config_file(config_file, primary_config, logger):
+    """
+    Read simulation configuration file with details on shower
+    and array simulations
+
+    Attributes
+    ----------
+    config_file: str
+        Name of simulation configuration file
```

Comment:
I have a colleague who would remark on the missing `.` at the end of the sentence (and in other places).

Reply:
There are some crazy people out there... 
Fixed.